src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1677,36 +1676,80 @@ public void VerifyXmlResolver(bool provideResolver)
                 catch
                 {
                 }
-
-                ((IDisposable)listener).Dispose();
             }
 
             static async Task ProcessRequests(
-                HttpListener listener,
-                Action<HttpListenerRequest> requestReceived,
+                TcpListener listener,
+                Action requestReceived,
                 CancellationToken cancellationToken)
             {
+                const string Response =
+                    "HTTP/1.1 200 OK\r\nContent-Type: text/plain; charset=UTF-8\r\nContent-Length: 0\r\n\r\n";
+
                 while (!cancellationToken.IsCancellationRequested)
                 {
-                    HttpListenerContext ctx;
+                    Socket socket;
 
                     try
                     {
-                        ctx = await listener.GetContextAsync();
+#if NETCOREAPP
+                        socket = await listener.AcceptSocketAsync(cancellationToken);
+#else
+                        socket = await listener.AcceptSocketAsync();
+#endif
```

Comment:
I tried that, and our stylecop settings got angry (zero argument invocations should have the close paren on the same line).  Rather than a suppression and a `#if` it seemed cleaner to just do it this way.